src/components/cities-index.js

```diff
@@ -0,0 +1,62 @@
+import { useSelector } from "react-redux";
+import { fetchCities } from "../actions";
+import { Sparklines, SparklinesLine, SparklinesReferenceLine } from 'react-sparklines';
+import SearchValue from "./search-form.js";
+import _ from 'lodash';
+
+const CitiesIndex = () => {
+  const cities = useSelector((state) => state.cities);
+
+  function renderCities() {
+    if (!_.isEmpty(cities)) {
+      let resultTemp = cities.stats.map(t => t.temperature);
+      let resultPressure = cities.stats.map(t => t.pressure);
+      let resultHumidity = cities.stats.map(t => t.humidity);
+
+      return (
+        <tr >
+          <td style={{ height: '100px' }} className="align-middle"><strong>{cities.name}</strong></td>
+          <td><Sparklines data={cities.stats.map((city => city.temperature))} svgHeight={75}>
+            <SparklinesLine color="blue" />
+            <SparklinesReferenceLine type="mean" />
+          </Sparklines><p>{Math.round(_.mean(resultTemp))} F</p></td>
+          <td><Sparklines data={cities.stats.map((city => city.pressure))} svgHeight={75}>
+            <SparklinesLine color="green" />
+            <SparklinesReferenceLine type="mean" />
+          </Sparklines><p>{Math.round(_.mean(resultPressure))} hPa</p></td>
+          <td><Sparklines data={cities.stats.map((city => city.humidity))} svgHeight={75}>
+            <SparklinesLine color="red" />
+            <SparklinesReferenceLine type="mean" />
+          </Sparklines><p>{Math.round(_.mean(resultHumidity))} %</p></td>
+        </tr>
+      )
+    }
+    return <tr>
+      <td>n/a</td>
+      <td>n/a</td>
+      <td>n/a</td>
+      <td>n/a</td>
+    </tr>
+  }
+
+  return (
+    <div className="col text-center">
+      <SearchValue fetchCities={fetchCities} />
```

Comment:
`SearchValue` makes it sound like the component is a value itself - `SearchForm` or something like it would probably be better

Reply:
You're passing `fetchCities` but `SearchValue` isn't using it (as it shouldn't b/c it has access directly to Redux)